src/services/beacon/utils.ts

```diff
@@ -10,7 +10,7 @@ export const ALICE_PRIV_KEY = "edsk3QoqBuvdamxouPhin7swCvkQNgq4jP5KZPbwWNnwdZpSp
 
 export const rpcNodes: Record<Network, string> = {
   mainnet: "https://mainnet.api.tez.ie",
```

Comment:
Let's do the env thing for mainnet as well.

Reply:
@Man-Jain done.